composer.json

```diff
@@ -15,12 +15,12 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "~4.0|~5.0|~6.0|~7.0|~8.0|~9.0",
```

Comment:
do not remove 9.0.. just add 10.0 as well

Reply:
done